source/mapping-rules/mapping-rule-options/mapping-rule-filters.txt

```diff
@@ -0,0 +1,132 @@
+.. _rm-mapping-filters:
+
+====================
+Mapping Rule Filters
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In a migration project, you can use mapping rule javascript expressions to filter mapping rules based on document values. Mapping rule filters are available to all mapping rule types.
```

Comment:
Can we call these "Mapping rule filters" consistently? While they do use JavaScript expressions, we should use this as a description and not the name of the feature.